package.json

```diff
@@ -85,40 +85,40 @@
     ]
   },
   "dependencies": {
-    "@commitlint/cli": "7.5.0",
+    "@commitlint/cli": "7.5.2",
     "@commitlint/config-conventional": "7.5.0",
     "@oclif/command": "1.5.x",
-    "@oclif/config": "1.12.4",
+    "@oclif/config": "1.12.8",
     "@oclif/plugin-help": "2.1.6",
     "@semantic-release/commit-analyzer": "6.1.x",
     "@semantic-release/git": "7.0.8",
     "@semantic-release/github": "5.2.10",
     "@semantic-release/npm": "5.1.4",
     "@semantic-release/release-notes-generator": "7.1.x",
-    "@types/jest": "23.3.13",
-    "@types/node": "10.12.21",
-    "cli-ux": "5.0.0",
-    "commitizen": "3.0.5",
+    "cli-ux": "5.2.0",
+    "commitizen": "3.0.7",
     "cz-conventional-changelog": "2.1.x",
     "husky": "1.3.1",
     "inquirer": "6.2.2",
-    "jest": "24.0.0",
-    "lint-staged": "8.1.1",
-    "lodash": "4.x.x",
+    "lint-staged": "8.1.4",
     "rimraf": "2.6.3",
     "semantic-release": "15.13.3",
     "shelljs": "0.8.x",
-    "ts-jest": "23.10.5",
     "tslib": "1.9.3",
-    "tslint-config-stoplight": "1.2.x",
     "typedoc": "0.14.2"
   },
   "devDependencies": {
-    "@oclif/dev-cli": "1.21.0",
+    "@oclif/dev-cli": "1.21.x",
     "@types/inquirer": "0.0.x",
-    "@types/lodash": "4.14.120",
-    "@types/shelljs": "0.8.2",
-    "ts-node": "8.0.2",
-    "typescript": "3.3.1"
+    "@types/jest": "24.0.x",
+    "@types/lodash": "4.14.x",
+    "@types/node": "11.9.x",
+    "@types/shelljs": "0.8.x",
+    "jest": "24.1.x",
```

Comment:
Shouldn't be in dependencies? If I am not mistaken, dev dependencies are trimmed out during the build process.